test/Microsoft.TestPlatform.TestUtilities/TestPluginCacheHelper.cs

```diff
@@ -54,8 +54,10 @@ public static void SetupMockExtensions(string[] extensions, Action callback, Moc
 
             mockFileHelper.Setup(fh => fh.DirectoryExists(It.IsAny<string>())).Returns(true);
 
-            var testableTestPluginCache = new TestableTestPluginCache(extensions.ToList());
-            testableTestPluginCache.Action = callback;
+            var testableTestPluginCache = new TestableTestPluginCache(extensions.ToList())
```

Comment:
Using object initializer is IMHO less readable than the original code. And populating multiple members in initializer leads to errors that are harder to debug than if object initializer is not used. This would probably have benefit only if we used `init;`.